I0328 23:45:53.615902       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Role/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:53.619161       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ServiceAccount/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:53.637620       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create RoleBinding/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:53.641801       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Deployment/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:53.648155       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create Service/test-cluster-discovery for controller TidbCluster/test-cluster successfully
I0328 23:45:53.654266       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd in  test-cluster successful
I0328 23:45:53.659459       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-pd-peer in  test-cluster successful
I0328 23:45:53.662836       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-pd-3731616 for controller TidbCluster/test-cluster successfully
I0328 23:45:53.668876       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22773", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-pd in  test-cluster successful
I0328 23:45:53.678022       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:45:53.678050       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
I0328 23:45:53.805025       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:45:53.805063       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
I0328 23:45:53.891934       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:45:54.747513       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
I0328 23:45:54.773948       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:45:54.774018       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:45:54.831231       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
, service acto-namespace/test-cluster-pd has no endpoints
E0328 23:45:54.837875       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:46:02.846116       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Error response 503 URL http://test-cluster-pd.acto-namespace:2379/pd/health,body response: no leader
E0328 23:46:02.855813       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:46:21.679276       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
E0328 23:46:21.691690       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:46:39.942795       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
E0328 23:46:39.962367       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed tidbcluster: [acto-namespace/test-cluster]'s pd status sync failed, can not to be upgraded, requeuing
E0328 23:46:45.033680       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
I0328 23:46:45.064594       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:46:45.064635       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for PD cluster running, requeuing
E0328 23:46:45.130231       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.50.109:2379: connect: connection refused
I0328 23:46:45.143937       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22888", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tikv-peer in  test-cluster successful
I0328 23:46:45.148233       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22888", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tikv-3831336 for controller TidbCluster/test-cluster successfully
I0328 23:46:45.154911       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"22888", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tikv in  test-cluster successful
I0328 23:46:45.193515       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:46:45.193557       1 tidb_cluster_controller.go:124] TidbCluster: acto-namespace/test-cluster, still need sync: TidbCluster: [acto-namespace/test-cluster], waiting for TiKV cluster running, requeuing
E0328 23:46:50.251322       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: net/http: request canceled (Client.Timeout exceeded while awaiting headers)
I0328 23:46:55.330081       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:55.330138       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: net/http: request canceled (Client.Timeout exceeded while awaiting headers), requeuing
E0328 23:46:55.397586       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.50.109:2379: connect: connection refused
I0328 23:46:55.452607       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
E0328 23:46:55.452672       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.50.109:2379: connect: connection refused, requeuing
E0328 23:46:55.528052       1 pd_member_manager.go:190] failed to sync TidbCluster: [acto-namespace/test-cluster]'s status, error: Get http://test-cluster-pd.acto-namespace:2379/pd/health: dial tcp 10.96.50.109:2379: connect: connection refused
E0328 23:46:55.540312       1 tidb_cluster_controller.go:126] TidbCluster: acto-namespace/test-cluster, sync failed Get http://test-cluster-pd.acto-namespace:2379/pd/api/v1/stores: dial tcp 10.96.50.109:2379: connect: connection refused, requeuing
I0328 23:47:16.747948       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"23205", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb-peer in  test-cluster successful
I0328 23:47:16.759829       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"23205", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create Service test-cluster-tidb in  test-cluster successful
I0328 23:47:16.763665       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"23205", FieldPath:""}): type: 'Normal' reason: 'Successfully Create' create ConfigMap/test-cluster-tidb-6662316 for controller TidbCluster/test-cluster successfully
I0328 23:47:16.770114       1 event.go:282] Event(v1.ObjectReference{Kind:"TidbCluster", Namespace:"acto-namespace", Name:"test-cluster", UID:"9015c38a-bbbd-4521-a6b8-6bf14b914350", APIVersion:"pingcap.com/v1alpha1", ResourceVersion:"23205", FieldPath:""}): type: 'Normal' reason: 'SuccessfulCreate' create StatefulSet test-cluster-tidb in  test-cluster successful
I0328 23:47:17.737796       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:17.856761       1 equality.go:87] Service spec diff for acto-namespace/test-cluster-tidb:   v1.ServiceSpec{
  	Ports: []v1.ServicePort{
  		{
  			... // 3 identical fields
  			Port:       4000,
  			TargetPort: {IntVal: 4000},
- 			NodePort:   0,
+ 			NodePort:   31460,
  		},
  		{
  			... // 3 identical fields
  			Port:       10080,
  			TargetPort: {IntVal: 10080},
- 			NodePort:   0,
+ 			NodePort:   30060,
  		},
  	},
  	Selector:  {"app.kubernetes.io/component": "tidb", "app.kubernetes.io/instance": "test-cluster", "app.kubernetes.io/managed-by": "tidb-operator", "app.kubernetes.io/name": "tidb-cluster"},
  	ClusterIP: "",
  	... // 12 identical fields
  }
I0328 23:47:17.856799       1 tidb_member_manager.go:442] Sync TiDB service acto-namespace/test-cluster-tidb, spec equal: false, annotations equal: true, label equal: true
I0328 23:47:17.911837       1 service_control.go:88] update Service: [acto-namespace/test-cluster-tidb] successfully, kind: , name: test-cluster
I0328 23:47:17.937108       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
E0328 23:47:18.113836       1 pod_control.go:185] failed to update pod acto-namespace/test-cluster-tidb-1 with cluster labels map[app.kubernetes.io/component:tidb app.kubernetes.io/instance:test-cluster app.kubernetes.io/managed-by:tidb-operator app.kubernetes.io/name:tidb-cluster controller-revision-hash:test-cluster-tidb-788bdc6cfc statefulset.kubernetes.io/pod-name:test-cluster-tidb-1 tidb.pingcap.com/cluster-id:7215726540687462063], TidbCluster: test-cluster, err: Operation cannot be fulfilled on pods "test-cluster-tidb-1": the object has been modified; please apply your changes to the latest version and try again
E0328 23:47:18.716168       1 pod_control.go:185] failed to update pod acto-namespace/test-cluster-tidb-0 with cluster labels map[app.kubernetes.io/component:tidb app.kubernetes.io/instance:test-cluster app.kubernetes.io/managed-by:tidb-operator app.kubernetes.io/name:tidb-cluster controller-revision-hash:test-cluster-tidb-788bdc6cfc statefulset.kubernetes.io/pod-name:test-cluster-tidb-0 tidb.pingcap.com/cluster-id:7215726540687462063], TidbCluster: test-cluster, err: Operation cannot be fulfilled on pods "test-cluster-tidb-0": the object has been modified; please apply your changes to the latest version and try again
I0328 23:47:19.167619       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:19.331927       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:19.384988       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:19.686591       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:19.739673       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:20.886184       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:20.939925       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:22.088744       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:22.141441       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:23.286892       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:36.971565       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:36.997838       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:37.324910       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:37.382384       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully
I0328 23:47:38.525519       1 tidb_failover.go:53] tidb failover is disabled for acto-namespace/test-cluster, skipped
I0328 23:47:46.795656       1 tidbcluster_control.go:69] TidbCluster: [acto-namespace/test-cluster] updated successfully

